doc-Community_Collaboration_Guide/topics/introduction.adoc

```diff
@@ -1,14 +1,14 @@
 [[ccg-introduction]]
 == Introduction
 
-This guide has been provided by Red Hat to assist enterprise projects in collaborating with upstream communities on documentation. For Red Hat, this guide provides tips for evaluating a community to see if they are prepared for collaboration with an enterprise software team, communication guidelines, information about supported and unsupported tools, and guidelines for structuring git repositories.
+This guide has been provided by https://www.redhat.com/[Red Hat] to assist enterprise projects in collaborating with upstream communities on documentation. For Red Hat, this guide provides tips for evaluating a community to see if they are prepared for collaboration with an enterprise software team, communication guidelines, information about supported and unsupported tools, and guidelines for structuring git repositories.
```

Comment:
I think we should make it so that links to outside sites open in a new window. We can do this by putting a ^ before the ] and adding :linkattrs: to the attributes.

Reply:
@nengard Fine with me, feel free to do that in a separate PR (there are other external links in the doc that would need fixing). I'd like to get this PR merged first as I have follow-up changes depending on this.